packages/jobs/lib/runner/render.runner.ts

```diff
@@ -68,17 +103,17 @@ export class RenderRunner implements Runner {
                     image: { ownerId: ownerId, imagePath: `nangohq/nango-runner:${imageTag}` },
                     serviceDetails: { env: 'image' },
                     envVars: [
-                        { key: 'NODE_ENV', value: process.env['NODE_ENV'] },
-                        { key: 'NANGO_CLOUD', value: process.env['NANGO_CLOUD'] },
+                        { key: 'NODE_ENV', value: process.env['NODE_ENV'] || NodeEnv.Dev },
+                        { key: 'NANGO_CLOUD', value: process.env['NANGO_CLOUD'] || 'true' },
```

Comment:
I would default to false, because it would change the local development behavior
```suggestion
                        { key: 'NANGO_CLOUD', value: process.env['NANGO_CLOUD'] || 'false' },
```

Reply:
render runners are only used in the context of nango cloud and never locally

Reply:
ah true 👌🏻